supportedBackends/local/src/main/scala/cromwell/backend/impl/shadowlocal/ShadowLocalBackendLifecycleActorFactory.scala

```diff
@@ -0,0 +1,79 @@
+package cromwell.backend.impl.shadowlocal
+
+import java.nio.file.Path
+
+import cromwell.backend.BackendConfigurationDescriptor
+import cromwell.backend.sfs._
+import cromwell.backend.validation.DockerValidation
+import cromwell.core.PathFactory._
+import cromwell.core.PathWriter
+
+import scala.sys.process.Process
+
+class ShadowLocalBackendLifecycleActorFactory(override val configurationDescriptor: BackendConfigurationDescriptor)
+  extends SharedFileSystemBackendLifecycleActorFactory {
+  override def asyncJobExecutionActorClass = classOf[ShadowLocalAsyncJobExecutionActor]
+
+  override def supportsDocker = true
+}
+
+case class ShadowLocalJob(jobId: String) extends SharedFileSystemJob
+
+class ShadowLocalAsyncJobExecutionActor(override val params: SharedFileSystemAsyncJobExecutionActorParams)
+  extends SharedFileSystemAsyncJobExecutionActor[ShadowLocalJob] {
+
+  override def processArgs = {
+    val dockerRun = DockerValidation.optional.extract(validatedRuntimeAttributes) map buildDockerRunCommand getOrElse ""
+    new SharedFileSystemCommand(Seq("/bin/bash", "-c", s"cat ${jobPaths.script} | $dockerRun /bin/bash <&0"))
+  }
+
+  override lazy val stdoutSubmit = jobPaths.stdout.untailed
+
+  override lazy val stderrSubmit = jobPaths.stderr.untailed
+
+  override def getJob(process: Process, stdoutWriter: PathWriter, stderrWriter: PathWriter) = {
+    val p = ShadowLocalBackendLifecycleActorFactory.simpleProcessFieldP.get(process)
+    val pid = ShadowLocalBackendLifecycleActorFactory.unixProcessFieldPid.get(p)
```

Comment:
It's both.

Will reply to rest of comments later, but I _do_ dream of a local backend that can recover. 😉